### DISCUSSION TITLE
Help me

### DISCUSSION THREAD

**Discussion by DEVFermium** [Authenticator-Extension/Authenticator#1023] [General]
I linked my Roblox account with the authentication tool. I reset my computer a while ago, and my authentication app and recovery key were all deleted. So I can't log in to my Roblox account. I tried contacting Roblox about this problem, but Roblox doesn't respond.
The grammar may be awkward using Google Translate.

**Comment by Ananna270** [comment]
Oh no, that sounds frustrating! Have you tried reaching out to any other resources for assistance with your Roblox account? Maybe there are some online communities or forums that could offer some advice. Or, if you're looking for professional help, you might consider checking out a platform like [Any Help Now](https://app.anyhelpnow.com/search/services), where you can connect with people who offer tech support services. Hope you're able to resolve the issue soon!
